src/catalog/src/builtin.rs

```diff
@@ -6130,14 +6120,6 @@ ON mz_catalog.mz_materialized_views (id)",
     is_retained_metrics_object: false,
 };
 
-pub const MZ_CLUSTER_LINKS_IND: BuiltinIndex = BuiltinIndex {
-    name: "mz_cluster_links_ind",
-    schema: MZ_INTERNAL_SCHEMA,
-    sql: "IN CLUSTER mz_introspection
-ON mz_internal.mz_cluster_links (cluster_id)",
-    is_retained_metrics_object: false,
-};
```

Comment:
Do we need to inform console about this in case they were using this index?

Reply:
Will follow up with them––this won't make it out until next week's release so plenty of time. Thank you for the heads up